providers/infrastructure-vsphere/v1.5.1/cconly/base.yaml

```diff
@@ -2100,3 +2103,56 @@ spec:
             {{- range .additionalFQDN }}
             - {{ . }}
             {{- end }}
+  - name: systemWideProxy
```

Comment:
I tested the new format on this branch, since I can't push to your branch
https://github.com/vmware-tanzu/tanzu-framework/compare/topic/lubron/system-wide-proxy?expand=1

Reply:
Thanks, updated!